ynr/apps/sopn_parsing/helpers/parse_tables.py

```diff
@@ -264,6 +272,12 @@ def parse_raw_data_for_ballot(ballot):
                     data = data.drop(data.index[i])
                 except IndexError:
                     break
+        else:
```

Comment:
Need a refresher on what this is fixing - might it be not necessary since Sym's changes?

Reply:
Ah ok I _think_ I understand